php/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM centos:6
+
+MAINTAINER Eric Poe <eric.poe@gmail.com
```

Comment:
I guess that proves that the contents of "MAINTAINER" are completely arbitrary. Good to know!